gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -8,49 +8,46 @@ import React from 'react';
 
 import { NxCheckbox } from '@sonatype/react-shared-components';
 import { NxRadio } from '@sonatype/react-shared-components';
-
-import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
-import { faCalendar } from '@fortawesome/free-solid-svg-icons';
+import { NxButton } from '@sonatype/react-shared-components';
+import { NxStatefulTextInput } from '@sonatype/react-shared-components';
 
 export default function NxFormLayoutExample() {
+  function validator(val: string) {
+    return val.length ? null : 'Must be non-empty';
+  }
+
   return (
     <form className="nx-form">
       <div className="nx-form-row">
         <div className="nx-form-group">
```

Comment:
Looking through the DOM makes me ask, what exactly is the point of `.nx-form-group`?  Does it ever contain anything other than exactly one `.nx-label`?

Reply:
> Looking through the DOM makes me ask, what exactly is the point of `.nx-form-group`? Does it ever contain anything other than exactly one `.nx-label`?

There are some instances in IQ where a form control doesn't have a label, and there may be others elsewhere.

Reply:
> There are some instances in IQ where a form control doesn't have a label, and there may be others elsewhere.

Does RSC support that? How? `NxTextInput` always has a label and we no longer document how to create text inputs without the react component.

Reply:
> NxTextInput always has a label and we no longer document how to create text inputs without the react component.

Nevermind, I don't know where I got the idea that that was true, it isn't.

Reply:
>Nevermind, I don't know where I got the idea that that was true, it isn't.

We should probably demonstrate how text fields without labels fit into form layouts

Reply:
> There are some instances in IQ where a form control doesn't have a label, and there may be others elsewhere.

I guess I also question whether we really want that in our design language.  There are certainly places where a standalone text input makes sense (search boxes, table filters, etc), but I don't know that there's any case _in a form_ where you'd want an unlabeled text input.  Maybe if it's a form that consists of only a single text input and a submit button?

Reply:
> > There are some instances in IQ where a form control doesn't have a label, and there may be others elsewhere.
> 
> I guess I also question whether we really want that in our design language. There are certainly places where a standalone text input makes sense (search boxes, table filters, etc), but I don't know that there's any case _in a form_ where you'd want an unlabeled text input. Maybe if it's a form that consists of only a single text input and a submit button?

I'm not sure either, but the initial question was about whether we could dispense with the wrapping `.nx-form-group` since `<label>` appears to do the same job, but I think that we are better off with the extra wrapping `<div>` and the flexibility that gives us. I think asking `<label>` to also be our form group wrapper gives it too much responsibility within the DOM.